pycbc/results/psd.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2022
+# Tito Dal Canton, Gareth Cabourn Davies, Ian Harry
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the
+# Free Software Foundation; either version 3 of the License, or (at your
+# option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General
+# Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+
+#
+# =============================================================================
+#
+#                                   Preamble
+#
+# =============================================================================
+#
+"""
+Module to generate PSD figures
+"""
+import matplotlib
+matplotlib.use('agg')
```

Comment:
Do not set matplotlib's backend in a module, it will mess up some use cases as soon as somebody imports (a module that imports) `pycbc.results`.